pkg/nodeconfig/nodeconfig.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/openshift/windows-machine-config-operator/pkg/ignition"
```

Comment:
Consider splitting the changes into two commits; 1) the refactoring and 2) Add check for file copying on azure